docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,26 @@
 # Staging release notes
 
+## April 2024
+
+**11 April 2024**
+
+Frontend version: 1.0.0-20240328.0943 | Backend version: 1.97.2-20240405.1708
+
+
+| Type | Change | Description |
+| ---- | ------ | ----------- |
+| **Feature** | SAML certificates expiry notifications | The `notify-expiring-saml-certs` job has been enhanced to email tenant admins about SAML certificate expirations. Notifications will exclude certificates for applications not aligned with TechPass standards. |
```

Comment:
I don't think we need to mention the job name.

Reply:
i removed this alr
 @JLeeChan